KalturaWowzaServer/src/main/java/com/kaltura/media_server/services/Utils.java

```diff
@@ -231,7 +231,14 @@ public static String getStreamName(IMediaStream stream) {
     }
 
     public static String getMediaServerHostname() throws IOException, InterruptedException {
-        Process p = Runtime.getRuntime().exec("hostname -f");
+        return getMediaServerHostname(true);
+    }
+
+    public static String getMediaServerHostname(boolean full) throws IOException, InterruptedException {
+        String command = "hostname";
+        if (full)
```

Comment:
Full should be true by default.
Leaving this as is changed the default behavior we previously had, and if I am not mistaken all media servers are registered to the DB with the full host names so this may cause register request to fail.